.modules/service/ecs.tf

```diff
@@ -56,4 +56,4 @@ resource "aws_ecs_task_definition" "task" {
       }
     }, var.container_definitions)
   ])
-}
\ No newline at end of file
+}
```

Comment:
Revert this change.

Reply:
Reverted. But as it was auto-applied by the formatter, I think it should be modified in the future.